src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -6212,15 +6212,16 @@ protected function addCopy(&$script)
      * objects.
      *
      * @param  boolean \$deepCopy Whether to also copy all rows that refer (by fkey) to the current row.
+     * @param  boolean \$makeNew Whether to reset autoincrement PKs and make the object new.
      * @return ".$this->getObjectClassName(true)." Clone of current object.
      * @throws PropelException
      */
-    public function copy(\$deepCopy = false)
+    public function copy(\$deepCopy = false, \$makeNew = true)
```

Comment:
New arguments should behave like the old one, so default should be false.

Reply:
```suggestion
    public function copy(\$deepCopy = false, \$makeNew = false)
```

This would improve backwards compatibility and prevent the need for a major.